confignode/src/main/java/org/apache/iotdb/confignode/manager/load/heartbeat/HeartbeatWindow.java

```diff
@@ -48,4 +48,10 @@ public void addHeartbeat(HeartbeatPackage newHeartbeat) {
       }
     }
   }
+
+  public long getLastSendTime() {
```

Comment:
remove the HeartBeatWindow class, move List<HeartbeatPackage> to HeartbeatCache class

Reply:
Fixed.